perfkitbenchmarker/disk.py

```diff
@@ -172,6 +172,8 @@ class BaseDiskSpec(spec.BaseSpec):
     mount_point: None or string. Directory of mount point.
     num_striped_disks: int. The number of disks to stripe together. If this is
         1, it means no striping will occur. This must be >= 1.
+    os_type: None or string. OS type of the VM to which the disk will be
```

Comment:
For now I'm okay just getting rid of the `os_type` stuff. You can make `GetDeviceId` check if `self.disk_number` is defined and return the DeviceId if it is.